### PR TITLE
CA-258357: Modified the build script to build the MSI installer

### PR DESCRIPTION
**Pull request by GaborApatiNagy**
Signed-off-by: Gabor Apati-Nagy <gabor.apati-nagy@citrix.com>

### CODE REVIEW

File: mk/package-and-sign.sh [hunk 1]
```diff
@@ -64,8 +63,8 @@ echo "INFO: Build and sign the installers..."
 . ${REPO}/mk/build-installers.sh
 
 #collect output and extra files to the OUTPUT_DIR
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${OUTPUT_DIR}
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.l10n.exe ${OUTPUT_DIR}
+cp ${WIX}/out${BRANDING_BRAND_CONSOLE}/${BRANDING_BRAND_CONSOLE}.msi ${OUTPUT_DIR}
```
Comment (MihaelaStoica) [inline mk/package-and-sign.sh]: These files are already copied the the OUTPUT_DIR at the end of build-installers.sh script. We should only do it once.

File: mk/xenadmin-build.sh [hunk 2]
```diff
@@ -156,22 +149,20 @@ cp ${REPO}/XenAdmin/bin/Release/{CommandLib.pdb,${BRANDING_BRAND_CONSOLE}.pdb,Xe
 
 ISO_DIR=${SCRATCH_DIR}/iso-staging
 mkdir_clean ${ISO_DIR}
-install -m 755 ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${ISO_DIR}/${BRANDING_BRAND_CONSOLE}Setup.exe
+install -m 755 ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE}.msi ${ISO_DIR}/${BRANDING_BRAND_CONSOLE}.msi
 cp ${REPO}/mk/ISO_files/AUTORUN.INF ${ISO_DIR}
 cp ${REPO}/Branding/Images/AppIcon.ico ${ISO_DIR}/${BRANDING_BRAND_CONSOLE}.ico
 #CP-18097
 tar cjf ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE}.installer.tar.bz2 -C ${ISO_DIR} .
-install -m 755 ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE}Setup.exe
```
Comment (MihaelaStoica) [inline mk/xenadmin-build.sh]: Why has this line (and line 174) been removed?

**Comment by MihaelaStoica** [comment]
@GaborApatiNagy Also the mk/ISO_files/AUTORUN.INF file needs to be updated

File: mk/xenadmin-build.sh (hunk 2 above)
Comment (GaborApatiNagy) [inline mk/xenadmin-build.sh]: This had only been a duplicate so I decided to remove it

**Comment by GaborApatiNagy** [comment]
Thanks for the comments Mihaela, I have fixed the PR

**Pull request merged**